src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -556,15 +591,19 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
   // If the warning acknowledgement is shown, we need to handle the submit differently because we might want to show the streaming acknowledgement
   const handleWarningAcknowledgementSubmit = useCallback(() => {
     if (activeQuote?.isStreaming) return setShouldShowStreamingAcknowledgement(true)
+    if ((activeQuote as ArbitrumBridgeTradeQuote)?.direction === 'withdrawal')
+      return setShouldShowArbitrumBridgeAcknowledgement(true)
     handleFormSubmit()
-  }, [handleFormSubmit, activeQuote?.isStreaming])
+  }, [activeQuote, handleFormSubmit])
 
   const handleTradeQuoteConfirm = useCallback(() => {
     if (isUnsafeQuote) return setShouldShowWarningAcknowledgement(true)
     if (activeQuote?.isStreaming) return setShouldShowStreamingAcknowledgement(true)
+    if ((activeQuote as ArbitrumBridgeTradeQuote)?.direction === 'withdrawal')
```

Comment:
Maybe we would prefer to use `isArbitrumBridgeTradeQuote(...)` than casting?

Reply:
Absolutely! 8fb16906b9